pkg/apis/logicalcluster/v1alpha1/logical-cluster.go

```diff
@@ -44,8 +44,8 @@ type LogicalCluster struct {
 
 // LogicalClusterSpec describes a cluster.
 type LogicalClusterSpec struct {
-	// ClusterProviderDescName is a reference to a ClusterProviderDesc resource
-	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="ProviderName is immutable"
+	// ClusterProviderDesc is a reference to a ClusterProviderDescName resource
```

Comment:
This is probably the source of the above issues

Reply:
fixed